doc/classes/Rect2i.xml

```diff
@@ -16,7 +17,7 @@
 		<constructor name="Rect2i">
 			<return type="Rect2i" />
 			<description>
-				Constructs a default-initialized [Rect2i] with default (zero) values of [member position] and [member size].
+				Constructs an empty [Rect2i] with its [member position] and [member size] set to [constant Vector2i.ZERO].
```

Comment:
Wouldn't "default" or "zero" be better than "empty"? To me "empty" implies some kind of container with contents, like an array or dictionary. Or maybe "empty" could mean "nothing is inside", but a zero rect has no insides. Or, another option is to just say we are constructing a Rect2i:

```suggestion
				Constructs a [Rect2i] with its [member position] and [member size] set to [constant Vector2i.ZERO].
```

Reply:
Many Variants in the code are initialized filled with `0`, or nothing, and they tend to check their "emptiness" with the same value, although I do agree it's a bit nonsensical. But `default` by itself doesn't say much, either. I accept the suggestion regardless.